src/ray/core_worker/lib/java/jni_utils.h

```diff
@@ -244,9 +244,10 @@ template <typename NativeT>
 inline void JavaListToNativeVector(
     JNIEnv *env, jobject java_list, std::vector<NativeT> *native_vector,
     std::function<NativeT(JNIEnv *, jobject)> element_converter) {
-  int size = env->CallIntMethod(java_list, java_list_size);
   RAY_CHECK_JAVA_EXCEPTION(env);
   native_vector->clear();
+  if (java_list == nullptr) return;
```

Comment:
Because somewhere I use this function to convert Java ObjectIds to C++ ObjectId, however, when passing by value, there are no ids to convert. In that case, I would pass a nullptr.

Reply:
That's find but you need to move `RAY_CHECK_JAVA_EXCEPTION(env);` under `int size = env->CallIntMethod(java_list, java_list_size);`.

Reply:
This modification has been reverted.